html-block-and-inline/html-block-and-inline/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+  </head>
+  <body>
+    <h1><b>Blue Americana Snuggie &reg; Blanket</b></h1>
+    <h2><b>by SnuggieStore</b></h2>
+    <p>
+      Price: <s>$19.99</s> $14.99 <br/>
```

Comment:
instead of the `br` tags I would recommend using `p` tags to break each line of text. `br` tags are more suited for poetry or other layouts which require a very specific text format.